cypress/e2e/BiographyTestsFrontEnd/index.js

```diff
@@ -0,0 +1,62 @@
+/// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+Then('page title is {string}',(title)=>{
+cy.get(`h1:contains("${title}")`).should('be.visible').and('include.text',title)
+})
+
+And('the individuals title is {string}',(chiefCommunicationTeam)=>{
```

Comment:
I will mention this here and also in a general feedback - parameter's name should be specific enough ( e.g. not just str or param) but not too granular - I will explain using this step and please refactor name of  parameters in the rest of the step definitions.
the individuals title is {string} - you named the string param "chiefCommunicationTeam" - well, you should not care what the feature scenario passes in, for you its just a string which happens to be some sort of a title , well name it "title" or "individualTitle", dont just use the value of param to name it